data/json/mutations.json

```diff
@@ -2181,8 +2181,22 @@
     "cancels": [ "SMELLY", "SMELLY2" ],
     "threshreq": [ "THRESH_PLANT", "THRESH_ELFA" ],
     "category": [ "PLANT", "ELFA" ],
+    "changes_to": [ "ROSEBUDS" ],
     "social_modifiers": { "lie": 10 }
   },
+  {
+    "type": "mutation",
+    "id": "ROSEBUDS",
+    "name": "Rosebuds",
+    "points": 3,
+    "visibility": 10,
+    "ugliness": -6,
+    "prereqs": [ "FLOWERS" ],
+    "description": "The top of your head is blooming with rosebuds.  They're eye-catching, and have a strong fragrance that makes you pleasant to be around.",
```

Comment:
```suggestion
    "description": "The top of your head is blooming with rosebuds.  They're eye catching, and have a strong fragrance that makes you pleasant to be around.",
```

Reply:
I think this is proper grammar, I'm not sure.

Reply:
You usually only need to hyphenate compound adjectives if they come before whatever they're describing: "the rosebuds are eye catching" vs "the eye-catching rosebuds": https://www.quickanddirtytips.com/education/grammar/how-to-use-a-hyphen

To use an example from a mistake I made in a previous PR: "A super concentrated mutagen." Is it a (super) (concentrated) mutagen, a mutagen that's both super and concentrated, or is it a (super concentrated) mutagen? The ambiguity doesn't exist when it comes after: "A mutagen that's super concentrated." You couldn't write two adjectives in sequence that way.

Reply:
Wowie, I thought I was good at English. Thanks, corrected.